src/lm_buddy/buddy.py

```diff
@@ -0,0 +1,41 @@
+from lm_buddy.integrations.wandb import ArtifactLoader, WandbArtifactLoader
+from lm_buddy.jobs._entrypoints import run_finetuning, run_lm_harness, run_prometheus, run_simple
+from lm_buddy.jobs.common import EvaluationResult, FinetuningResult, SimpleResult
+from lm_buddy.jobs.configs import (
+    EvaluationJobConfig,
+    FinetuningJobConfig,
+    LMHarnessJobConfig,
+    PrometheusJobConfig,
+    SimpleJobConfig,
+)
+
+
+class LMBuddy:
+    """Your buddy in the (L)LM space.
+
+    Simple wrapper around executable functions for tasks available in the library.
+    """
+
+    def __init__(self, artifact_loader: ArtifactLoader = WandbArtifactLoader()):
```

Comment:
I wonder..if Artifact is a WandB specific term, does it make sense to call the more abstract loader a ModelLoader or similar for clarity?

Reply:
Artifact is a general term used by the 3 main tracking services (WandB, MlFlow, Comet). But I do think we will need to iterate on the purpose of this class in the near future.

I was looking at how other frameworks (e.g, HF `transformers`) handles tracking across multiple different tracking services. They implement a call-back based approach with something like a `WandbTrackingCallback` and `CometTrackingCallback`, etc..  I think we will want to abstract this interface to handle artifact/tracking stuff for the various frameworks eventually, but right now, I don't have a better option without a substantial bit of work.